plugins/modules/resource_info.py

```diff
@@ -77,7 +77,7 @@
 - debug:
     var: result.resources[0].value
 
-- name: "Read all Registries (Katello)"
+- name: "Read all Registries"
```

Comment:
```suggestion
- name: "Read all Container Registries"
```
Certainly only if that would be correct...

Reply:
To be honest, I have no idea, and thus would not touch it :D